CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+1.6.6
+-----
+
```

Comment:
Consider adding a note explaining what #698 means for the end user. Something like:

```
This is the first release as a single multi-pex, meaning the released Pex may be ran with multiple different Python interpreter versions, rather than needing a distinct release for each interpreter.
```

Reply:
Isn't that redundant on the releases page where there will be a single pex binary available for download which says the same thing by inference?

Reply:
Erm. I guess this is also displayed - wonkily - on pypi where it isn't redundant. Will fix.

Reply:
Fixed.